git-hooks/pre-push

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env php
+
+<?php
+echo "Running npm run lint...";
+exec('npm run lint', $output, $returnCode);
+if (count($output) !== 8) { // Hard coded array length to 8 since return code is empty
```

Comment:
I might do
```
if(!strstr(implode(' ', $output) , "0 problems")) {
...
}
```
because its possible you have an error that takes up exactly 8 lines.  Not a big deal though.  Go ahead and merge if it works.